src/thorin/analyses/schedule.cpp

```diff
@@ -174,6 +174,8 @@ const CFNode* Scheduler::schedule_smart(const PrimOp* primop) {
         }
     }
 
+    if (primop->isa<Enter>() || Enter::is_out_mem(primop) || Enter::is_out_frame(primop)) result = late;
+
```

Comment:
Actually, I want to remove this check in line 152. The reason for having them was to make LLVM happy. If we don't get any performance regressions we could remove this check.

Reply:
This was some code I left around. It's removed in latest commits (see commit aa66650cc42ffa3b9595ddabf8395ea3c7017bdb). There is still a check though, a couple of lines above this. I suggest to keep this for the time being, so that we can iron the issues with the pe_fix branch first before merging.

Reply:
Yes, I saw your commit. I was referring to the code a couple of lines above this. Your commit just reminded me of the issue.